src/instrumentation/vercel/anthropic.ts

```diff
@@ -0,0 +1,390 @@
+import { LANGTRACE_ADDITIONAL_SPAN_ATTRIBUTES_KEY } from '@langtrace-constants/common'
+import {
+  APIS,
+  Event,
+  FrameworkSpanAttributes,
+  LLMSpanAttributes,
+  Vendors
+} from '@langtrase/trace-attributes'
+import {
+  context,
+  Exception, Span, SpanKind,
+  SpanStatusCode,
+  trace,
+  Tracer
+} from '@opentelemetry/api'
+
+import { calculatePromptTokens, estimateTokens } from '@langtrace-utils/llm'
+import { addSpanEvent } from '@langtrace-utils/misc'
+
+export async function generateTextPatchAnthropic (
+  this: any,
+  args: any[],
+  originalMethod: (...args: any[]) => any,
+  method: string,
+  tracer: Tracer,
+  langtraceVersion: string,
+  sdkName: string,
+  version?: string
+): Promise<(...args: any[]) => any> {
+  let url: string
+  if (args[0]?.model?.config?.baseURL !== undefined) {
+    url = args[0]?.model?.config?.baseURL
+  } else {
+    url = this?._client?.baseURL
+  }
+
+  const customAttributes = context.active().getValue(LANGTRACE_ADDITIONAL_SPAN_ATTRIBUTES_KEY) ?? {}
+  const attributes: FrameworkSpanAttributes & LLMSpanAttributes = {
+    'langtrace.sdk.name': sdkName,
+    'langtrace.service.name': Vendors.VERCEL,
+    'langtrace.service.type': 'framework',
+    'langtrace.service.version': version,
+    'gen_ai.operation.name': 'chat',
+    'langtrace.version': langtraceVersion,
+    'gen_ai.request.model': args[0]?.model?.modelId,
+    'url.full': '',
+    'url.path': '',
```

Comment:
i have faced the same issue before with `url.full` & `url.path` being empty, empty values hurt otel conventions but we can let it pass for now

Reply:
yes, saw it discard this comment then